### ISSUE TITLE
Brush cursor offset when using scroll bars

### ISSUE THREAD

**Issue by IAmNotSJ** [drawpile/Drawpile#1321]
When attempting to use the vertical scroll bar to reposition the view of the canvas, the brush cursor dot will rapidly be offset to the left and back to the position that the tablet pen is placed. This additionally results in the cursor drawing a line while using the scroll bar. When letting go of the scroll bar, the brush cursor will remain offset until moved off of the canvas into the ui, where it will return to the tablet pen

I'm using a Kamvas Pro 24

Attached is the video of the bug in progress

https://github.com/drawpile/Drawpile/assets/84604061/91b234db-57cb-4678-9813-4d7dff771914

**Comment by askmeaboutlo0m** [comment]
Odd, I don't have an idea off-hand why this would be happening. Some troubleshooting to narrow it down.

Which version of Drawpile are you using? Help → About Drawpile shows it.

Which version of Windows are you using and are you using some kind of shell extensions (classic start menu, mods to the styling or similar) on it? If so, does the issue go away if you disable those?

Which tablet driver do you have enabled in Drawpile? Edit → Tablet Driver has a few different options, not all of which necessarily work with your tablet. Which one(s) does the issue show up with?

If you switch to [the continuous release](https://github.com/drawpile/Drawpile/releases/tag/continuous), is the problem still present?

And lastly, in the continuous release, if you go into Edit → Preferences and in the General tab switch Renderer to either "Hardware (experimental)" or "Software (experimental)" (doesn't matter which one) and restart Drawpile, is the problem still present? Those new renderers have a different kind of UI setup, which might incidentally fix this oddity.

**Comment by IAmNotSJ** [comment]
When switching tablet drivers, the issue was fixed, but here's all of the info that I had before switching if its useful

Version is 2.2.1
Windows 10 22H2 with no shell extensions
Tablet driver is KisTablet Windows Ink (Issue was fixed when tested with any other tablet driver)
Switching to the continuous release did not fix the issue, nor did switching the renderer

**Comment by askmeaboutlo0m** [comment]
Alright, then I guess it is just down to a driver mismatch. I tried getting it to happen with my Wacom tablet on Windows 10, but couldn't manage to do, no matter the combination of Wacom and Drawpile driver settings.

There's not much we can do about tablets not cooperating, but I added it [to the various driver issue points in the FAQ](https://docs.drawpile.net/help/common/faq#why-does-the-brush-position-detach-from-my-cursor), which at least documents the problem.

Thanks for reporting it!
